### ISSUE TITLE
Insert in HTMLEditor do not allow for changing size / setting alignment

### ISSUE THREAD

**Issue by NobrainerWeb** [unclecheese/silverstripe-kickassets#23]
First of all - so nice to see the Browse functionality (upload, move and all from wysiwyg) - something that has been missing for way to long in SilverStripe.

When adding an image via the Browse Files... button, there is no way to define size or alignment.
To do this you then have to open the normal image insert modal - seems a bit cumbersome.

**Comment by unclecheese** [comment]
Yeah, I may add that soon. I just didn't think anyone ever used that feature, since it's so easy to just size it with the mouse in the wysiwyg editor.

**Comment by adamjudd** [comment]
Another vote for this - less the sizing (for the reason you noted @unclecheese), but more the positioning options.
